### DISCUSSION TITLE
GRUB?

### DISCUSSION THREAD

**Discussion by edwinm** [minios-linux/minios-live#11] [General]
So I wanted to try MiniOS and burned it to an USB stick and booted from it.

"Fast. Simple. Reliable." Sounds good.

Then I was unpleasantly welcomed by GRUB. A whole list of commands it could type.

Ah, there I see it: "boot", let's type that.

Oh, I first have to load the kernel. How do I do that? No instructions whatsoever.

Ah, I see "exit". Let's exit GRUB and continue booting in MiniOS.

Oh. It boots Windows.

"Fast. Simple. Reliable."

I'm not sure about Fast and Reliable. But I'm sure it's not Simple.

**Comment by crim50n** [comment]
You should use the instructions to install the system on the drive, then you won't have any problems. If the menu does not appear, then you have installed the system incorrectly.
https://github.com/minios-linux/minios-live/wiki/Install-MiniOS-on-hard-disk-or-USB-flash-drive

**Reply by edwinm** [reply]
I followed the instructions on the website https://minios.dev/en/#downloads

I flashed the iso to USB drive, like what is common for many distributons.That should just work, right?

**Reply by crim50n** [reply]
If you are unable to install MiniOS using the methods that apply to most distributions, you should rely on the method described in the documentation.
